### PR TITLE
Add wrappers for Qualimap

### PR DESCRIPTION
**Pull request by wm75**
FOR CONTRIBUTOR:
* [x] - I have read the [CONTRIBUTING.md](https://github.com/galaxyproject/tools-iuc/blob/master/CONTRIBUTING.md) document and this tool is appropriate for the tools-iuc repo.
* [x] - License permits unrestricted use (educational + commercial)
* [x] - This PR adds a new tool or tool collection
* [ ] - This PR updates an existing tool or tool collection
* [ ] - This PR does something else (explain below)

### CODE REVIEW

**Comment by wm75** [comment]
supersedes #2494

File: tools/qualimap/qualimap_bamqc.xml [hunk 1]
```diff
@@ -0,0 +1,474 @@
...
+                label="Dataset specifying regions" />
+                <param argument="-os" name="outside_stats" type="boolean" truevalue="--outside-stats" falsevalue="" checked="false"
+                label="Invert regions"
+                help="If selected, report read statistics *outside* the regions in the regions file." />
+            </when>
+        </conditional>
+        <param argument="-oc" name="per_base_coverage" type="boolean" truevalue="--output-genome-coverage" falsevalue="" checked="false"
+        label="Generate per-base coverage output"
+        help="Produce additional tabular output listing the coverage at every site (omitting only zero-coverage positions) in the selected regions of the genome. Caution: Will generate a huge dataset for anything but small input genomes or restricted regions!" />
+        <param argument="--skip-duplicated/--skip-dup-mode" name="duplicate_skipping" type="select" display="checkboxes" multiple="true" optional="true"
```
Comment (bgruening) [inline tools/qualimap/qualimap_bamqc.xml]: to keep argument useful for the future we should only use a valid argument ... e.g. only one of them

**Comment by wm75** [comment]
@zipho I finally got round to look at qualimap in more detail. I started with your WIP #2494, but given how much I ended up extending it, I took the liberty to open a new PR.
Thanks a lot for starting this though.

File: tools/qualimap/qualimap_bamqc.xml [hunk 2]
```diff
@@ -0,0 +1,474 @@
...
+            <param argument="-nw" name="n_bins" type="integer" value="400"
+            label="Number of bins to use in across-reference plots"
+            help="Affected plots: Coverage, Mapping Quality and Insert Size across reference, Mapped reads GC-content distribution; the value determines the resolution of the affected plots. Note: The lower the value, the higher the memory usage of the tool!" />
+            <param argument="-c" name="paint_chromosome_limits" type="boolean" truevalue="--paint-chromosome-limits" falsevalue="" checked="true"
+            label="Draw chromosome limits"
+            help="Affected plots: Coverage, Mapping Quality and Insert Size across reference; in across-reference plots, indicate chromosome boundaries with dotted lines and labels" />
+            <param argument="-gd" name="genome_gc_distr" type="select"
+            label="Plot expected GC-content distribution of the following reference genome"
+            help="Affected plot: Mapped reads GC-content distribution; include a precalculated GC-content distribution for the selected (Qualimap-supported) reference genome in the plot">
+                <option value="">None</option>
```
Comment (bgruening) [inline tools/qualimap/qualimap_bamqc.xml]: optional="true" instead of None?

Comment (wm75) [reply tools/qualimap/qualimap_bamqc.xml]: yes, makes sense.

File: tools/qualimap/qualimap_bamqc.xml (hunk 1 above)
Comment (wm75) [reply tools/qualimap/qualimap_bamqc.xml]: agreed

**Comment by bgruening** [comment]
Merci!

**Pull request merged**